charts/stable/metube/values.yaml

```diff
@@ -15,9 +17,9 @@ service:
         targetPort: 8081
 
 persistence:
-  downloads:
+  config:
     enabled: true
-    mountPath: "/downloads"
+    mountPath: /config
 
```

Comment:
Add a download key also here and in the GUI.
So it's just easy to switch to hostpath or whtaever from the GUI so people dont start looking around to find the dir.

Reply:
can the gui option have a variable change the mountpath?